persistence/pre_persistence/proto/account.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package utility;
+option go_package = "/utility/test/";
+
+message Account {
+  bytes Address = 1;
+  string Amount = 2;
+}
+
+message Pool {
```

Comment:
Can you add a one line comment (if my understanding is correct): `// The in memory database will store a pool for every Account in the network keyed by it's Name`.

What is the name here?